src/data.js

```diff
@@ -0,0 +1,53 @@
+export const IngredientItemsLeft = [
```

Comment:
I know this is probably a lot of moving parts, but you don't have to capitalize your arrays for data.  Usually this is camelCase, whereas, components are Capitalized.